<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BembelDbug</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="naolist" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>525</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>525</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="nao_edit_container">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="nao_edit_tabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_config">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <attribute name="title">
          <string>Configuration</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
        <widget class="QWidget" name="tab_calibration">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <attribute name="title">
          <string>Calibration</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="sync_widget_container" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="nao_ssh_eth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="text">
             <string>ssh (eth)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nao_ssh_wlan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="text">
             <string>ssh (wlan)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionssh_all_eth0"/>
    <addaction name="actionssh_all_wlan0"/>
    <addaction name="actionadd_nao"/>
    <addaction name="actionclear_added_naos"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_remember_tools"/>
    <addaction name="action_remember_geometry"/>
    <addaction name="action_mdi_mode"/>
    <addaction name="action_clear_autostart"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionFindNao">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPlayer_Positions">
   <property name="text">
    <string>&amp;Player Positions</string>
   </property>
  </action>
  <action name="actionssh_all_eth0">
   <property name="text">
    <string>ssh all (eth0)</string>
   </property>
  </action>
  <action name="actionssh_all_wlan0">
   <property name="text">
    <string>ssh all (wlan0)</string>
   </property>
  </action>
  <action name="action_remember_tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remember open tools</string>
   </property>
  </action>
  <action name="action_remember_geometry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remember window geometry</string>
   </property>
  </action>
  <action name="action_clear_autostart">
   <property name="text">
    <string>Clear autostart tools</string>
   </property>
  </action>
  <action name="action_mdi_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Force single window mode</string>
   </property>
   <property name="toolTip">
    <string>Always start in single window mode</string>
   </property>
  </action>
  <action name="actionAdd_Nao">
   <property name="text">
    <string>Add Nao</string>
   </property>
  </action>
  <action name="actionadd_nao">
   <property name="text">
    <string>add nao</string>
   </property>
  </action>
  <action name="actionclear_added_naos">
   <property name="text">
    <string>clear added naos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
